seed.js

```diff
@@ -0,0 +1,22 @@
+const faker = require('faker');
+const Product = require('./models/products.model')
+const initDB = require('./db')
+
+require('dotenv').config()
+initDB();
```

Comment:
Just curious to know why you are initializing DB twice once in the index.js file and another one here.

Reply:
This was just some stand alone code I wrote to populate the products collection. Not used as part of the app.